klasses/factories.py

```diff
@@ -37,6 +38,14 @@ class BootcampRunFactory(DjangoModelFactory):
     novoed_course_stub = LazyAttribute(
         lambda bootcamp_run: "-".join(bootcamp_run.title.lower().split(" "))[0:20]
     )
+    bootcamp_run_id = LazyAttribute(
+        lambda bootcamp_run: "bootcamp-v1:{type}+{topic}-{format}+R{run}".format(
+            type=random.choice(["public", "private"]),
+            topic="".join([word[0] for word in bootcamp_run.title.split()]).upper(),
+            format=random.choice(["ol", "f2f"]),
+            run=bootcamp_run.run_key,
+        )[0:255]
```

Comment:
why this substring indexing is required?

Reply:
@umarmughal824 I added this substring indexing to make sure that we never get/generate a string with a length more than 255 hence failing the model field constraint of  255 length & failing the tests along the way since it's used in factories. Also, since we are not taking care of the exact format ATM, so I thought just the first 255 characters should do us fine without any other validations. 

Let me know if you think otherwise and I'll make the change.

Reply:
FYI, the tests doesn't fail without this indexing.

Reply:
Anyhow merge it

Reply:
Okay